change-notes/1.23/analysis-javascript.md

```diff
@@ -11,6 +11,7 @@
   - [rate-limiter-flexible](https://www.npmjs.com/package/rate-limiter-flexible)
 
 * The call graph has been improved to resolve method calls in more cases. This may produce more security alerts.
+* Promises derived from a Deferred object are now recognized.
```

Comment:
```suggestion
```
We have just improved a query that was introduced in 1.23, so no change note is needed.